mautic/mautic
# The preference center should list and check marked all the new global categories

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🟢 
| New feature/enhancement? (use the a.x branch)      | 🔴
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🟢 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 


## Description

Contact category preference subscriptions are not visible.

For the new contact (or any existing contacts that haven't unsubscribed from categories), the preferences center boxes should be checked, and all available categories should appear in the contact's Category preferences.

### 📋 Steps to test this PR:
1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Create categories
3. Create a contact
4. Look at the contact category preferences - as of now, you'll see that they're empty
5. Create a segment with that contact in it
6. Create a preference center page with the Categories preference block
7. Create a segment-type email, assign it to the category you've created, and attach the preference page. Please make sure to include the unsubscribe link.
8. Send the email and click the preference/unsubscribe link.
